pages/linux/dir.md

```diff
@@ -0,0 +1,24 @@
+# dir
+
+> Dir is equivalent to ls -C -b; that is, by default files are listed in columns, sorted vertically, and special characters are represented by backslash escape sequences.
+> More information: <https://manned.org/dir>.
+
+- Displays all the hidden files along with two files denoted by "." and "..":
+
+`dir -a`
+
+- Displays author of all the files. -l is required to display the contents in the form of a list:
```

Comment:
I think you should add the `dir -l` example to explain that argument before `-author`
```suggestion
- List files including the author (`-l` is required):
```